master/docs/relnotes/index.rst

```diff
@@ -21,6 +21,7 @@ Features
 ~~~~~~~~
 
 * :class:`GitPoller` now has a ``buildPushesWithNoCommits`` option to allow the rebuild of already known commits on new branches.
+* Add GitLab authentication plugin for web UI. See :class:`buildbot.www.oauth2.GitLabAuth`.
```

Comment:
No. The next release note is created at the time of release.

Le mar. 23 févr. 2016 03:36, Vladimir Rutsky <notifications@github.com> a
écrit :

> In master/docs/relnotes/index.rst
> <https://github.com/buildbot/buildbot/pull/2010#discussion_r53729314>:
>
> > @@ -21,6 +21,7 @@ Features
> >  ~~~~~~~~
> >
> >  * :class:`GitPoller` now has a ``buildPushesWithNoCommits`` option to allow the rebuild of already known commits on new branches.
> > +* Add GitLab authentication plugin for web UI. See :class:`buildbot.www.oauth2.GitLabAuth`.
>
> @tardyp <https://github.com/tardyp> I believe this should go into
> 0.9.0b8.rst release notes.
>
> I created template (and filled it with my stuff) in PR #2012
> <https://github.com/buildbot/buildbot/pull/2012>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buildbot/buildbot/pull/2010/files#r53729314>.
>